keps/sig-cloud-provider/20180530-cloud-controller-manager.md

```diff
@@ -352,7 +352,7 @@ common integration hooks may be added
 The development of the
 [Cloud Controller Manager](https://github.com/kubernetes/kubernetes/tree/master/cmd/cloud-controller-manager)
 and
-[Cloud Provider Interface](https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/cloud.go)
+[Cloud Provider Interface](https://github.com/kubernetes/cloud-provider/blob/master/cloud.go)
```

Comment:
So this is the right place from which to consume the interface. It is actually just a replica right now but I think that's fine.

Reply:
thanks